guide/popular-topics/webhooks.md

```diff
@@ -0,0 +1,209 @@
+
+```js
+const webhookClient = new Discord.WebhookClient('id', 'token');
+```
+
+## Creating webhooks
+
+### Creating webhooks through server settings
+
+You can create webhooks directly through the discord client, go to Server Settings, you will see a `Webhooks` tab.
+
+![Webhook tab](~@/images/creating-webhooks-1.png)
+
+Once you are there, click on the `Create Webhook` button on the top right. This will create a webhook, from here you can edit the channel, the name, and the avatar. Copy the link, the first part is the id, and the second is the token.
+
+![Creating a Webhook](~@/images/creating-webhooks-2.png)
+
+### Creating webhooks with discord.js
+
+Discord.js provides a method for creating webhooks called <branch version="11.x" inline>[`TextChannel.createWebhook()`](https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=createWebhook)</branch><branch version="12.x" inline>[`TextChannel.createWebhook()`](https://discord.js.org/#/docs/main/master/class/TextChannel?scrollTo=createWebhook)</branch>.
+
+```js
+channel.createWebhook('Snek', 'https://i.imgur.com/IgscS45.jpg')
+	.then(webhook => console.log(`Created webhook ${webhook}`))
+	.catch(console.error);
+```
+
+## Editing webhooks
+
+<branch version="11.x">
+
+You can edit Webhooks and WebhookClients to change their avatar and name using [`Webhook.edit()`](https://discord.js.org/#/docs/main/stable/class/Webhook?scrollTo=edit) and [`WebhookClient.edit()`](https://discord.js.org/#/docs/main/stable/class/WebhookClient?scrollTo=edit).
+
+```js
+webhook.edit('Snek', 'https://i.imgur.com/IgscS45.jpg')
+	.then(webhook => console.log(`Edited webhook ${webhook}`))
+	.catch(console.error);
+```
+
+</branch>
+<branch version="12.x">
+
+You can edit Webhooks and WebhookClients to change their name, avatar, and channel using [`Webhook.edit()`](https://discord.js.org/#/docs/main/master/class/Webhook?scrollTo=edit).
+
+```js
+webhook.edit({
+	name: 'snek',
+	avatar: 'https://i.imgur.com/IgscS45.jpg',
+	channel: '222197033908436994',
+}).then(webhook => console.log(`Edited webhook ${webhook}`)).catch(console.error);
+```
+
+</branch>
+
+## Using webhooks
+
+Webhooks, unlike bots, can send more than one embed per message, up to 10. They can also send attachments and normal content. The <branch version="11.x" inline> [`Webhook.send()`](https://discord.js.org/#/docs/main/stable/class/Webhook?scrollTo=send)</branch><branch version="12.x" inline>[`Webhook.send()`](https://discord.js.org/#/docs/main/master/class/Webhook?scrollTo=send)</branch> method to send to a webhook is very similar to the method for sending to a text channel. Webhooks can also change their name and avatar when sending a message.
```

Comment:
> Webhooks can also change their name and avatar when sending a message.

Not quite accurate, they can send an avatar and name to be used for this specific request, yes, it does not really "change" them in any way, webhooks just accept these parameters in the payload instead of inferring them from the logged in client, like the case for "standard" messages